### PR TITLE
Add shared models library collaborate

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: src/bundles/Collaborate/components/Code/Code.js [hunk 1]
```diff
@@ -9,39 +14,49 @@ class Code extends React.Component {
         var win = window.open(url, '_blank');
         win.focus();
     }
-
+    handleSubmit(val) {
+        console.log(val);
```
Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: remove?

File: src/bundles/Collaborate/components/Code/Code.js [hunk 2]
```diff
@@ -99,21 +98,30 @@ class Code extends React.Component {
...
                     </div>
 
+                    <div>
+                        <article role="main" styleName="what">
+                            <h2 style={{
+                                textAlign: "center"
+                            }}>What features will help your team? </h2>
+                            <p style={{textAlign: "center", fontSize: "20px"}}>
+                                Let us know what you would like to see in the library<br/>
+                                <a href="mailto:marketplace@digital.gov.au?subject=Feedback"
```
Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: /contact-us

File: src/bundles/Collaborate/components/Code/Code.js [hunk 3]
```diff
@@ -99,21 +98,30 @@ class Code extends React.Component {
 
                     </div>
 
+                    <div>
+                        <article role="main" styleName="what">
+                            <h2 style={{
+                                textAlign: "center"
+                            }}>What features will help your team? </h2>
+                            <p style={{textAlign: "center", fontSize: "20px"}}>
```
Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: put in css?

File: src/bundles/Collaborate/components/Code/Code.js [hunk 4]
```diff
@@ -61,7 +76,8 @@ class Code extends React.Component {
                                                 <a rel="external" target="_blank" href={item.url}
                                                    onClick={(e) => e.stopPropagation()}>{item.name}</a>
```
Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: why e.stopPropagation()?

File: src/bundles/Collaborate/components/Code/Code.js [hunk 5]
```diff
@@ -9,39 +14,49 @@ class Code extends React.Component {
...
+                        <article className="col-md-7 col-sm-12 col-md-push-1" styleName="code-text">
+
+
+                            <h2> New to open source?</h2>
+                            You can reduce cost, effort and risk by re-using tried and tested open source projects to build digital services.<br/>
+<br/>
+                            Making source code open is criteria 8 in the <a href="https://www.dta.gov.au/standard/" rel="external" target="blank">Digital Service Standard</a>
+                            You can learn how to <a href="https://opensource.guide/" rel="external" target="blank">launch and grow your project</a> or contact us if you would like some help.<br/>
+
+                        <a href="mailto:marketplace@digital.gov.au?subject=Open source projects"
```
Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: /contact-us

File: src/bundles/Collaborate/components/Code/Code.js (hunk 3 above)
Comment (maxious) [inline src/bundles/Collaborate/components/Code/Code.js]: 👍

File: src/bundles/Collaborate/components/Code/Code.js (hunk 4 above)
Comment (maxious) [inline src/bundles/Collaborate/components/Code/Code.js]: React has an "interesting" way of handling events such that they propagate up the DOM ie. this link is inside another thing that can be clicked on so they both get click events.

Comment (matt-sm) [inline src/bundles/Collaborate/components/Code/Code.js]: ahh - synthetic events and both have on click - gotcha

**Pull request merged**